T/t8code/build_tarballs.jl

```diff
@@ -0,0 +1,112 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+using Base.BinaryPlatforms
+const YGGDRASIL_DIR = "../.."
+include(joinpath(YGGDRASIL_DIR, "platforms", "mpi.jl"))
+
+name = "t8code"
+version = v"1.1.2"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/DLR-AMR/t8code/releases/download/v$(version)/t8code_v$(version).tar.gz",
+                  "0bd4bee6694735d14fb4274275fb8c4bdeacdbd29b257220c308be63e98be8f7"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd t8code/
+atomic_patch -p1 "${WORKSPACE}/srcdir/patches/mpi-constants.patch"
+
+# Set default preprocessor and linker flags
+# Note: This is *crucial* for Windows builds as otherwise the wrong libraries are picked up!
+export CPPFLAGS="-I${includedir}"
+export LDFLAGS="-L${libdir}"
+export CFLAGS="-O3"
+export CXXFLAGS="-O3"
+
+# Set necessary flags for FreeBSD
+if [[ "${target}" == *-freebsd* ]]; then
+  export LIBS="-lm"
+fi
+
+# Set necessary flags for Windows and non-Windodws systems
+FLAGS=()
+if [[ "${target}" == *-mingw* ]]; then
+  # Set linker flags only at build time (see https://docs.binarybuilder.org/v0.3/troubleshooting/#Windows)
+  FLAGS+=(LDFLAGS="$LDFLAGS -no-undefined")
+  # Configure does not find the correct Fortran compiler
+  export F77="f77"
+  # Link against ws2_32 to use the htonl function from winsock2.h
+  export LIBS="-lmsmpi -lws2_32"
+  # Disable MPI I/O on Windows since it causes p4est to crash
+  mpiopts="--enable-mpi --disable-mpiio"
+  # Linker looks for libmsmpi instead of msmpi, copy existing symlink
+  cp -d ${libdir}/msmpi.dll ${libdir}/libmsmpi.dll
+else
+  # Use MPI including MPI I/O on all other platforms
+  export CC="mpicc"
+  export CXX="mpicxx"
+  mpiopts="--enable-mpi"
+fi
+
+# Run configure
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-static --without-blas ${mpiopts}
+
+# Build & install
+make -j${nproc} "${FLAGS[@]}"
+make install
+
+# On Windows: copy DLLs to make them findable by BinaryBuilder
+if [[ "${target}" == *-mingw* ]]; then
+  cp ${libdir}/libp4est*.dll ${libdir}/libp4est.dll
+  cp ${libdir}/libsc*.dll ${libdir}/libsc.dll
+  cp ${libdir}/libt8*.dll ${libdir}/libt8.dll
+fi
```

Comment:
What's the problem?  If libraries have different names on different products, just list them: https://docs.binarybuilder.org/stable/reference/#BinaryBuilderBase.LibraryProduct.  You shouldn't move files around.

Reply:
Thank for the link, I've fixed it in 9c4a82d3a5f6c84796a671d391f08676874ef204.